contracts/0.4.24/Lido.sol

```diff
@@ -999,10 +790,65 @@ contract Lido is ILido, StETH, AragonApp {
     }
 
     /**
-    * @dev Size-efficient analog of the `auth(_role)` modifier
-    * @param _role Permission name
-    */
+     * @dev Size-efficient analog of the `auth(_role)` modifier
+     * @param _role Permission name
+     */
     function _auth(bytes32 _role) internal view auth(_role) {
         // no-op
     }
+
+    function _transferToStakingRouter(uint256 _maxDepositsCount) internal {
+        address stakingRouter = getStakingRouter();
+        require(stakingRouter != address(0), "STAKING_ROUTER_ADDRESS_ZERO");
+
+        uint256 buffered = _getBufferedEther();
```

Comment:
Just transfer `_getBufferedEther ` to StakingRouter

Reply:
fixed